adbc.h

```diff
@@ -315,6 +354,16 @@ struct ADBC_EXPORT AdbcError {
 ///
 /// \see AdbcConnectionGetInfo
 #define ADBC_INFO_DRIVER_ARROW_VERSION 102
+/// \brief The driver ADBC API version (type: int64).
+///
+/// The value should be one of the ADBC_VERSION constants.
```

Comment:
Is it allowed to be a later version?

Reply:
"Yes", in that if/when there's another revision, then that would be a valid value, but earlier clients/drivers of course wouldn't know about that.